src/css/relay.css

```diff
@@ -47,7 +24,10 @@
   --relayGrey90: #0c0c0d;
   --relayInk: #20123a;
   --relayInk70: #363959;
-  --relayMetropolis: "Metropolis", sans-serif, "SF Pro Text", "Segoe UI", Roboto, "Open Sans";
+  --relayMetropolisBold: "Metropolis Bold", "SF Pro Text", "Segoe UI", Roboto, "Open Sans", sans-serif;
```

Comment:
Because of how the fontFace is being set via JS — we needed separate font references for each weight class.

Reply:
These changes to the font paths were changed to solve for #148 (in commit https://github.com/mozilla/fx-private-relay-add-on/pull/249/commits/7e6054af414dbedbf2cd446807e9e054296ba776) — we were setting the font path relatively (https://github.com/mozilla/fx-private-relay-add-on/blob/main/src/css/relay.css#L5) and when we injected an object into a webpage (via content script) it was looking for the font on THEIR resources, which wasn't happening. 

To solve this, I'm using the [`FontFace()`](https://developer.mozilla.org/en-US/docs/Web/API/FontFace/FontFace) API to set the fonts in use. Another constraint of that is that I need to set each weight so I needed to have separate names since we're loading 4 individual fonts. 

The new flow is: 
- User loads element from the add-on (First run page, clicks on content script icon, opens panel) 
- `setCustomFonts()` function (from utils.js) runs, checking (via [this method](https://developer.mozilla.org/en-US/docs/Web/API/FontFaceSet/check)) if the fonts are loaded. If they're already loaded, nothing happens. If not, it loads them.

Reply:
Thank you!